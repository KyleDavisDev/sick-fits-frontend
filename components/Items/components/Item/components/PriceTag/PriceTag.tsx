import * as React from "react";
import { StyledSpan } from "./PriceTagStyle";
import formatMoney from "../../../../../../lib/formatMoney";

interface IPriceTagProps {
  children: number;
}

const PriceTag: React.FunctionComponent<IPriceTagProps> = props => {
  return <StyledSpan>{formatMoney(props.children)}</StyledSpan>;
};

export default PriceTag;
