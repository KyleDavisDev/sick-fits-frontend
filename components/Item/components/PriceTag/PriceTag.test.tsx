import * as React from "react";
import { shallow } from "enzyme";
import PriceTag from "./PriceTag";

describe("<PriceTag />", () => {
  it("renders", () => {
    const wrapper = shallow(<PriceTag>{0}</PriceTag>);

    expect(wrapper).toBeTruthy();
  });
});
