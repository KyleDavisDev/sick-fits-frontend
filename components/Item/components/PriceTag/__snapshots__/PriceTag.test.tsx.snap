// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PriceTag /> matches snapshot 1`] = `
<styled.span>
  $0
</styled.span>
`;
